TokenAcquirerTests/TokenAcquirer.cs

```diff
@@ -1,10 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Threading.Tasks;
```

Comment:
This was using the Implicit usings (which involve some tooling magic, which did not work in my VS when I got the rel/v2 branch from the source control)